doc/lua_api.txt

```diff
@@ -2725,6 +2726,37 @@ Some types may inherit styles from parent types.
     * noclip - boolean, set to true to allow the element to exceed formspec bounds. Default to false.
 * label, vertlabel
     * noclip - boolean, set to true to allow the element to exceed formspec bounds.
+    * horigin - Sets horizontal origin in relation to label text. Possible values:
+        * `left`: Left side of the bounding box
+        * `center`: Center of the bounding box
+        * `right`: Right side of the bounding box
+        * `leftline`: **Note**: Only for negative `newline_spacing`
+            * `label`: Acts like `left`
+            * `vertlabel`: Left end of first column
+        * `centerline`:
```

Comment:
I worry that the names of "xline" may be confusing, given it's more like the first character on the first line

Reply:
I could change the `horigin` values to `*column` since the first character can be considered a column.  For `vertlabel`s, it actually is the center of the entire column, not just the first character.  Making different values for `label` and `vertlabel` would just make the docs too verbose, I think.